nengo/conftest.py

```diff
@@ -18,8 +19,7 @@
 
 
 def pytest_configure(config):
-    rc.reload_rc([])
-    rc.set('decoder_cache', 'enabled', 'False')
+    matplotlib.use('AGG')
```

Comment:
In Matplotlib's list of backends (`matplotlib.rcsetup.non_interactive_bk`) it's listed as `u'agg'` (no capitals) so I would favor all lowercase here to be consistent.